Sources/Fluent/Query/Comparison.swift

```diff
@@ -11,6 +11,12 @@ extension Filter {
         case lessThanOrEquals
         case notEquals
     }
+
+    public enum PartialComparison {
```

Comment:
I think this would make more sense as a case on `Filter.Comparison`

```swift
enum Like {
     case anywhere, start, end
//...
case like(Like)
```